oktapam/utils/utils.go

```diff
@@ -113,3 +115,32 @@ func CreateCheckResourceDestroy(typeName string, checkResourceExists checkResour
 		return nil
 	}
 }
+
+// GenerateAttributeOverrides will evaluate which attributes on the resource may have overrides set,
```

Comment:
could you update the comment to include reasons why you would want to use this?

Reply:
I'm hoping [this](https://github.com/okta/terraform-provider-oktapam/pull/122/commits/990049a5bfe8ca59562ab23b24ad44d26779f7b5#diff-50ff6165be9dcbc6b7ac48292623314829fbdf124fba1537f184598594d087a9R120) may take care of all your questions around this.

I very much welcome feedback if you think there is a better way to deal with this.  I couldn't find much while searching about this kinda problem.

Reply:
Makes sense.  Thanks